cloudberry/zion/src/main/scala/edu/uci/ics/cloudberry/zion/actor/ProgressiveSolver.scala

```diff
@@ -105,7 +105,10 @@ class ProgressiveSolver(val dataManager: ActorRef,
             "end" -> JsNumber(boundary.getEnd().getMillis())
         ))
         // for query with slicing request, add current timeInterval information in its query results.
-        val infoValue = queryGroup.postTransform.transform(JsArray(returnedResult))
+        //val infoValue = queryGroup.postTransform.transform(JsArray(returnedResult))
+        val newresult = QueryPlanner.handleAvg(returnedResult)
```

Comment:
```suggestion
        val avgHandledResult = QueryPlanner.handleAvg(returnedResult)
```
Let's use more informative variable names.